### PR TITLE
Make it compatible with Erlang 19. Add travis

### PR DESCRIPTION
**Pull request by seriyps**
Just make it work on Erlang 19

### CODE REVIEW

File: src/epgsql_wire.erl [hunk 1]
```diff
@@ -209,6 +209,6 @@ encode_list(L) ->
     <<(byte_size(Bin)):?int32, Bin/binary>>.
 
 encode_standby_status_update(ReceivedLSN, FlushedLSN, AppliedLSN) ->
-    {MegaSecs, Secs, MicroSecs} = erlang:now(),
```
Comment (seriyps) [inline src/epgsql_wire.erl]: erlang:now() is deprecated http://erlang.org/doc/apps/erts/time_correction.html and it throws compilation warnings in 18 and 19 Erlang's
http://tryerl.seriyps.ru/#id=4df3

Comment (seriyps) [inline src/epgsql_wire.erl]: Here for example https://travis-ci.org/epgsql/epgsql/jobs/176440819#L328

Comment (davidw) [inline src/epgsql_wire.erl]: I figured as much.  I recall reading about some of the time changes - are you positive that that's the best call and not one of the monotonic ones?  It certainly works with all versions.

Comment (seriyps) [inline src/epgsql_wire.erl]: monotonic versions are added on erlang 18, but epgsql supports versions from 15 to 19. So, this call is the only one supported on all VMs. We can try to use conditional compilation, but I don't think there is a reason for this extra complexity.

**Comment by seriyps** [comment]
@davidw thanks!

**Pull request merged**